Jenkinsfile_k8s

```diff
@@ -242,12 +242,14 @@ pipeline {
                   // Execute build only for this matrix cell's setup
                   retry(count: 2, conditions: [kubernetesAgent(handleNonKubernetes: true), nonresumable()]) {
                     sh 'packer build -timestamp-ui -force -only="${PKR_VAR_image_type}.${PKR_VAR_agent_os_type}" ./'
+                    // adding manually a cpu architecture tag to the docker image
+                    sh 'docker tag jenkinsciinfra/jenkins-agent-${PKR_VAR_agent_os_type}:${PKR_VAR_architecture}'
```

Comment:
```suggestion
                    sh 'docker tag "jenkinsciinfra/jenkins-agent-${agent_type}:latest" "jenkinsciinfra/jenkins-agent-${PKR_VAR_agent_os_type}:${PKR_VAR_architecture}"'
```

- `docker tag` command requires 2 arguments: source image (e.g. an image already present in the output of `docker image ls` so it's the image created by `packer`) and new image name (with tag) to add as alias.

Reply:
at this point of the pipeline, can I use agent_type or should I remerge with the 2 variables ? : 
```
final String pkr_var_agent_os_type = agent_type.split('-')[0]
final String pkr_var_agent_os_version = agent_type.split('-')[1]
```